app/modules/users/models.py

```diff
@@ -580,6 +581,15 @@ def is_email_confirmed(self):
             return False
         return code.is_resolved
 
+    # creates email code and resolves it so above returns true
+    def bypass_email_confirmation(self):
+        import datetime
+
+        code = self.get_email_confirmation_code()
+        code.response = datetime.datetime.now()
```

Comment:
This records the timestamp but doesn't record the decision (technically an approval).  THis should be replaced with:
```python
from app.modules.auth.models import CodeDecisions
code = self.get_email_confirmation_code()
code.record(CodeDecisions.accept)
```
The `record` function includes a database merge.

Reply:
ah yes, this is much more solid.  thanks.